lib/web/desktop_playback.go

```diff
@@ -0,0 +1,246 @@
+/*
```

Comment:
Do we have a better package for this file than lib/web? I'm trying to avoid adding to the more "general" packages.

Reply:
I like that too, sounds good.